templates/worker/00-worker/ovirt/files/ovirt-disable-tx-checksum-offload.yaml

```diff
@@ -0,0 +1,8 @@
+filesystem: "root"
+mode: 0744
+path: "/etc/NetworkManager/dispatcher.d/pre-up.d/disable-tx-checksum-offload"
+contents:
+  inline: |
+    # This is a workaround for BZ#1794714
```

Comment:
Is there a way to check what is the default network, OpenshiftSDN or OVNkubernetes?
We use OpenshiftSDN by default and the fix is only relevant for OpenshiftSDN.

Reply:
I think the existence of `vxlan` interface means OpenShiftSDN. If that's true then all you need
is to check $1 name and only if its starts with `vxlan` then apply the change. 
We need to make sure this is really the case.

Reply:
or more reliably, `nmcli d show $1| awk '/GENERAL.TYPE:.*/ {print $2}'`

Reply:
I'll check it locally

Reply:
I checked and the device which is passed is the ethernet dev ens3, I think it should be available on both networks but I will check.
I'll ask the openshift network team what is the best way to distinguish between them

Reply:
I've checked and we can check if there is a file /var/lib/cni/bin/openshift-sdn that should do it